### DISCUSSION TITLE
希望添加“在potplayer中打开”功能

### DISCUSSION THREAD

**Discussion by Sissidark** [the1812/Bilibili-Evolved#3246] [功能建议 / Ideas]
douyuex，虎牙plus都有类似的功能，希望blblevvd也可以添加类似的功能，使得b站的直播和视频可以直接打开在potplayer中

**Comment by Geeyun-JY3** [comment]
#2120（不过好像不太一样）

**Reply by Sissidark** [reply]
主要是希望可以解析直播的链接，b站的使用f12找的在potplayer中也打不开
